packages/serverpod_test/lib/src/with_serverpod.dart

```diff
@@ -88,6 +109,12 @@ void Function(TestClosure<T>)
       });
 
       tearDownAll(() async {
+        if (setUpAllFailed) {
+          // If setUpAll failed, there is nothing to tear down.
+          // `transactionManager` might not be initialized so this callback is not safe to execute.
+          return;
+        }
```

Comment:
Question: Wouldn't it be better to make the transactionManager nullable? We may have done part of the setup here anyway if we get a throw on the pushSavePoint for example.

Reply:
I think you're right, this was a slight mistake on my end. In this case nullable is more suitable.